tools/maplint/lints/invalid_space_neighbors.yml

```diff
@@ -0,0 +1,6 @@
+/area/space:
+  banned_neighbors:
+  - /obj/machinery/door/airlock
+=/area/space:
```

Comment:
any reason not to move this up to all subtypes?

Reply:
Move what up?

Reply:
the structure check to all subtypes of space

Reply:
I'll add explicit checks for `/area/space/no_teleport` and `/area/space/nearstation` (the other two relevant types) but I can't use this for `/area/space/centcomm` because there are windows surrounding the holodeck scenes that I'm not touching

Reply:
Wait, none of those work, that's why I did that, fuck